snapshot/testsuite/testsuite.go

```diff
@@ -20,24 +20,26 @@ import (
 
 // SnapshotterSuite runs a test suite on the snapshotter given a factory function.
 func SnapshotterSuite(t *testing.T, name string, snapshotterFn func(ctx context.Context, root string) (snapshot.Snapshotter, func(), error)) {
-	t.Run("Basic", makeTest(t, name, snapshotterFn, checkSnapshotterBasic))
-	t.Run("StatActive", makeTest(t, name, snapshotterFn, checkSnapshotterStatActive))
-	t.Run("StatComitted", makeTest(t, name, snapshotterFn, checkSnapshotterStatCommitted))
-	t.Run("TransitivityTest", makeTest(t, name, snapshotterFn, checkSnapshotterTransitivity))
-	t.Run("PreareViewFailingtest", makeTest(t, name, snapshotterFn, checkSnapshotterPrepareView))
-	t.Run("Update", makeTest(t, name, snapshotterFn, checkUpdate))
-	t.Run("Remove", makeTest(t, name, snapshotterFn, checkRemove))
-
-	t.Run("LayerFileupdate", makeTest(t, name, snapshotterFn, checkLayerFileUpdate))
-	t.Run("RemoveDirectoryInLowerLayer", makeTest(t, name, snapshotterFn, checkRemoveDirectoryInLowerLayer))
-	t.Run("Chown", makeTest(t, name, snapshotterFn, checkChown))
-	t.Run("DirectoryPermissionOnCommit", makeTest(t, name, snapshotterFn, checkDirectoryPermissionOnCommit))
+	testutil.Parallel(t)
```

Comment:
It needs to apply at both level.

So for instance, the Basic test (the one that fails), is not run in parallel vis-a-vis of its sibling. 

If I remove this one, the SnapshotterSuite test will be sequential